listenbrainz/spotify_updater/spotify_read_listens.py

```diff
@@ -149,6 +149,9 @@ def make_api_request(user: dict, endpoint: str, **kwargs):
             spotipy_call = getattr(spotipy_client, endpoint)
             recently_played = spotipy_call(**kwargs)
             break
+        except (AttributeError, TypeError) as err:
+            current_app.logger.critical("Invalid spotipy endpoint or arguments:", err, exc_info=True)
```

Comment:
arguments to logger functions don't work like print. If you want to give an argument you need a % formatter placeholder in the string. this should be something like:

> current_app.logger.critical("Invalid spotipy endpoint or arguments: %s", err, exc_info=True)

However, do we even need to pass the exception if we also pass `exc_info=True`?

Reply:
Makes sense. Fixing.